spec/elastic_apm/rails_spec.rb

```diff
@@ -21,10 +21,22 @@
 
 if defined?(Rails)
   RSpec.describe Rails, :intercept do
+
+    before :all do
+      module RailsTestApp
+        class Application < Rails::Application
+        end
+      end
+    end
+
+    after(:all) do
+      Object.send(:remove_const, :RailsTestApp)
+    end
+
     describe '.start' do
       it 'starts the agent' do
         begin
-          ElasticAPM::Rails.start({})
+          ElasticAPM::Rails.start(RailsTestApp::Application.new, {})
```

Comment:
Can we move this to an int. test perhaps? Starting Rails will infect the `Rails` namespace for the whole process, so for cleanliness' sake it'd perhaps be beneficial to run it in its own process like the other int. tests.